hk2-locator/src/test/java/org/glassfish/hk2/tests/locator/negative/classanalysis/ConfigurablyBadClassAnalyzer.java

```diff
@@ -35,32 +35,32 @@
 @Singleton @Named(ConfigurablyBadClassAnalyzer.BAD_ANALYZER_NAME)
 public class ConfigurablyBadClassAnalyzer implements ClassAnalyzer {
     public static final String BAD_ANALYZER_NAME = "BadAnalyzer";
-    
+
     private boolean throwFromConstructor = false;
     private boolean throwFromMethods = false;
     private boolean throwFromFields = false;
     private boolean throwFromPostConstruct = false;
     private boolean throwFromPreDestroy = false;
-    
+
     private boolean nullFromConstructor = false;
     private boolean nullFromMethods = false;
     private boolean nullFromFields = false;
-    
-    @Inject @Named(ClassAnalyzer.DEFAULT_IMPLEMENTATION_NAME)
```

Comment:
Why was this changed?

Reply:
I will put it back, it seems I removed and now it selects the best candidate, which is the same.